py_ballisticcalc/drag_model.py

```diff
@@ -27,36 +26,48 @@ def __repr__(self):
 
 
 class DragModel:
-
-    def __init__(self, value: float,
+    """
+    :param BC: Ballistic Coefficient of bullet = weight / diameter^2 / i,
+        where weight is in pounds, diameter is in inches, and
+        i is the bullet's form factor relative to the selected drag model
+    :param drag_table: List of {Mach, Cd} pairs defining the standard drag model
+    :param weight: Bullet weight in grains
+    :param diameter: Bullet diameter in inches
+    :param length: Bullet length in inches
+    NOTE: .weight, .diameter, .length are only relevant for computing spin drift
+    """
+    def __init__(self, BC: float,
                  drag_table: typing.Iterable,
-                 weight: [float, Weight],
-                 diameter: [float, Distance]):
-        self.__post__init__(value, drag_table, weight, diameter)
+                 weight: [float, Weight]=0,
+                 diameter: [float, Distance]=0,
+                 length: [float, Distance]=0):
+        self.__post__init__(BC, drag_table, weight, diameter, length)
```

Comment:
looks strange, if it isn't a dataclass - it does not have a  __post_init__ method, just add all to __init__

the __post_init__ is a method that running automaticaly in dataclasses after __init__